packages/back-end/types/stats.d.ts

```diff
@@ -60,8 +60,15 @@ export type ExperimentMetricAnalysis = {
   }[];
 }[];
 
+export type BanditResponse = {
```

Comment:
todo: figure out how this gets persisted / consumed by GB.
- immediately update experiment model's variation weights, purge SDK cache
- probably needs to get piped to an event log or similar for weight changes (srm, maybe for vis)